.github/workflows/build_cmake.yml

```diff
@@ -0,0 +1,98 @@
+name: CI Build
+
+# Triggers on push and branches on the master
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+#env:
+
+# Allows to run this workflow manually from the Actions tab
+#workflow_dispatch:
+
+jobs:
+  debug:
```

Comment:
This is running only debug builds.

Recommend we do the following combination: 
debug+submodule
release+external (to test parsec install) 

That can be done in a second stage